tests/test_model.py

```diff
@@ -207,8 +207,8 @@ def test_segment_envivio_scte35_attribute():
 
 def test_segment_unknown_scte35_attribute():
     obj = m3u8.M3U8(playlists.CUE_OUT_INVALID_PLAYLIST)
-    assert obj.segments[0].scte35 is None
-    assert obj.segments[0].scte35_duration is None
+    assert obj.segments[0].scte35 == None
+    assert obj.segments[0].scte35_duration == None
```

Comment:
same here, I think you're mixing your PRs

Reply:
I believe it is the first commit, it has already been fixed in the second commit